src/docs/010_manual/20_install.adoc

```diff
@@ -89,32 +98,37 @@ To get started, you can run `tasks` as a first command:
 ----
 ./dtcw.ps1 tasks --group=doctoolchain
 ----
+
+NOTE: if you're making use of cmd.exe, please replace `.ps1` with `.bat`. Do this for subsequent commands.
+
```

Comment:
Line 102

NOTE: If using cmd.exe replace `ps1` with `.bat` for all subsequent commands.

**@Bennykillua Again, look for opportunities to make the wording more concise.**